docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,468 @@
+- **Opting out of the [Data Cache](#data-cache)**: This will fetch data on every request, which in turn will skip the Full Route Cache.
+- **Using the route segment config options `dynamic = force-dynamic` or `revalidate = 0`**: This will skip the Full Route Cache and the Data Cache. Meaning components will be rendered and data fetched on every request.
+
+> **Static vs. Dynamic Rendering**:
+>
+> You may see the terms **Static Rendering** and **Dynamic Rendering** being used. Static Rendering refers to caching the render output on the server, at build time, whereas Dynamic Rendering refers to rendering components on every request, at runtime.
+
+### Router Cache
+
+Next.js has an in-memory client-side cache that stores the React Server Component Payload, split by individual routes. This **Router Cache** is used to improve the navigation experience. Next.js [prefetches](/docs/app/building-your-application/routing/linking-and-navigating#prefetching) and caches routes that the user is likely going to navigate to (when using the `<Link >` component), and caches visited route segments, enabling [partial rendering](/docs/app/building-your-application/routing/linking-and-navigating#partial-rendering). This applies to newly visited routes, as well as backward/forward browser navigation.
+
+For the user, this results in [soft navigation](/docs/app/building-your-application/routing/linking-and-navigating#soft-navigation) (no page reloads) and faster page transitions; it also reduces unnecessary requests to the server.
+
+> **Good to know:**
+>
+> You may see the Router Cache being referred to as **Client-side Cache** or **Prefetch Cache**. While **Prefetch Cache** refers to the prefetched route segments, **Client-side Cache** refers to the whole Router cache, which includes both visited and prefetched segments.
+> This cache specifically applies to Next.js and Server Components, and is different to the browser's [bfcache](https://web.dev/bfcache/).
+
+#### Duration
+
+The cache is stored in the browser's temporary memory. Two factors determine how long the router cache lasts:
+
+- **Session**: The cache persists across [soft navigations](/docs/app/building-your-application/routing/linking-and-navigating#soft-navigation). However, it's cleared on page refresh.
+- **Automatic invalidation period**: The cache of an individual segment is automatically invalidated after a specific time. The duration depends on whether the route is [statically](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#static-rendering-default) or [dynamically](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#dynamic-rendering) rendered:
+  - **Dynamically rendered**: 30 seconds
+  - **Statically Rendered**: 5 minutes
+
+While a page refresh will clear **all** cached segments, the automatic invalidation period only affects the individual segment from the time it was last accessed or created.
```

Comment:
The actual caching behavior is more complex and depends on the `kind` set by the `prefetch` props of the `Link` component.
https://github.com/vercel/next.js/blob/05b2730f55371a7fb16b9afff6d01ad7010a897d/packages/next/src/client/components/router-reducer/get-prefetch-cache-entry-status.ts#L18-L37

Specifically, if **Statically Rendered** and `prefetch={true}`, the cache seems to expire in 30 minutes instead of 5 minutes.

Reply:
To be precise `prefetch={true}` is the default for static rendered routes. Adding `prefetch={true}` to a link that is dynamic rendered will opt-in to the 5 minutes cache. There is no 30 minutes cache, only a 30 seconds cache.


```suggestion
While a page refresh will clear **all** cached segments, the automatic invalidation period only affects the individual segment from the time it was last accessed or created.

By adding `prefetch={true}` or calling `router.prefetch` for a dynamically rendering route you can opt-in to the statically rendered behavior of caching for 5 minutes.
```